Core/DISET/RequestHandler.py

```diff
@@ -78,6 +79,23 @@ def _rh__initializeClass( cls, serviceInfoDict, lockManager, msgBroker, monitor
     cls.__monitor = monitor
     cls.log = gLogger
 
+    allNames = dir( cls )
+    exportedMethods = [ name for name in allNames if 'export_' in name ]
+    system, service = cls.__svcName.split( '/' )
+    setup = gConfig.getValue( 'DIRAC/Setup' )
+    systemSetup = gConfig.getValue( 'DIRAC/Setups/%s/%s' % ( setup, system ) )
+    result = gConfig.getOption( '/Systems/%s/%s/Services/%s/ResponseMonitoring/Methods' % ( system, systemSetup, service ), [] )
```

Comment:
Isn't the `getSeviceOption` function (defined in this same module) for exactly this purpose?